### ISSUE TITLE
Export/Import secrets from vault to/from yaml

### ISSUE THREAD

**Issue by counterbeing** [kontena/kontena#1633]
In the even that I ever wanted to spin up a new grid, be it for testing, or to replace production, it would be super useful to be able to grab all of my secrets. Additionally, there are other tools out there like Helm that use yaml config to describe variables. I think it might be really useful to have a yaml import and export to update all variables at once. 

It would also offer the piece of mind that if something happened to my grid, I could restore it quickly with a few commands.

**Comment by counterbeing** [comment]
I just noticed there's an issue #357 that resolves #354. #354 mentions the ability to export all secrets. Maybe this should stand as a feature request?

**Comment by kke** [comment]
Kind of duplicates #1625

**Comment by counterbeing** [comment]
I agree, it is partially, however that issue is requesting service specific secrets. I could see the functionality being listed under the same command. 

I just pushed a branch/PR. I tested it with my own kontena cluster, and everything looks good. If you guys like it, I can implement the `import` command as well, and update the docs.
